### PR TITLE
Add og-image for better branding

### LINKED ISSUE #24: Add an open graph meta tag to display logo of dotnetthailand
### Problem
When post a link from dotnetthailand website to other sites (ex. Facebook)
it'll display Gastby logo but it should be dotnetthailand logo instead.

image

### Solution
Add `<meta>` tag for `og:image` to every pgaes of the site.

```html
<meta property="og:image" content="url.to/the/image" />
```

REF: https://ogp.me/

### PR DESCRIPTION
**Pull request by surrealist**
All images designed by @MennStudio

Resolve #24

### CODE REVIEW

File: config/config.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ metadata:
   siteImage:
   favicon: "/assets/favicon.png"
   themeColor: "#ff0000"
+  ogImage: "/assets/dotnetthailand-ogimage-dark.png"
```
Comment (aaronamm) [inline config/config.yml]: @surrealist  Just curiosity, how the current system works without adding ogImage.
I wonder if we don't need to add ogImage here.

Comment (surrealist) [reply config/config.yml]: I pushed new commit with detailed explanation.

**Comment by surrealist** [comment]
I've tested on my local machine already, when inspect the web page I found this tag:

```html
<meta property="og:image" content="/assets/dotnetthailand-ogimage-dark.png" data-react-helmet="true">
```

I think this should be enough to complete the issue.

**Comment by aaronamm** [comment]
@surrealist Thanks for your contribution.

**Comment by aaronamm** [comment]
@surrealist  Thanks for your contribution.

**Pull request merged**